tests/integration/data_sources_and_expectations/test_expectation_conditions.py

```diff
@@ -242,6 +242,61 @@ def test_expect_column_min_to_be_between__pandas_row_condition(
     ),
 ]
 
+SPARK_TEST_CASES = SQL_TEST_CASES + [
+    pytest.param(
+        ComparisonCondition(
+            column=Column(name="name"), operator=Operator.EQUAL, parameter="albert"
+        ),
```

Comment:
For these integration test cases I was expecting to see the API we will document e.g. `Column("name") == "albert"` I'm not aware of anywhere else we are testing that.